inc/class-antivirus.php

```diff
@@ -579,18 +589,22 @@ public static function show_admin_menu() {
 				'checksum_verifier' => (int) ( ! empty( $_POST['av_checksum_verifier'] ) ),
 			);
 
+			$cron_enabled = $options['cronjob_enable'] || $options['safe_browsing'] || $options['checksum_verifier'];
+
 			// No cronjob?
-			if ( empty( $options['cronjob_enable'] ) ) {
+			if ( ! $cron_enabled ) {
```

Comment:
Also, here this context, I mean the variable is not necessary because you have created one point of check this via `_cron_enabled()`.

Reply:
see above